source/UF/Size-TruncatedConnected.lagda

```diff
@@ -0,0 +1,413 @@
+Ian Ray, 4th Febuary 2024.
+
+Modifications made by Ian Ray on 14 October 2024.
+
+Modifications made by Ian Ray on 7 January 2025.
+
+We develop some results that relate size, truncation and connectedness from
+the following paper:
+[1] Christensen, J.D. (2024), Non-accessible localizations.
+    Journal of Topology, 17: e12336.
+    https://doi.org/10.1112/topo.12336
+
+\begin{code}
+
+{-# OPTIONS --safe --without-K #-}
+
+open import UF.FunExt
+
+module UF.Size-TruncatedConnected
+        (fe : Fun-Ext)
+       where
+
+open import MLTT.Spartan hiding (_+_)
+open import Notation.CanonicalMap
+open import Notation.Decimal
+open import UF.ConnectedTypes fe
+open import UF.Embeddings
+open import UF.Equiv
+open import UF.EquivalenceExamples
+open import UF.PropTrunc
+open import UF.SmallnessProperties
+open import UF.Size
+open import UF.Subsingletons
+open import UF.Subsingletons-FunExt
+open import UF.Truncations fe
+open import UF.TruncationLevels
+open import UF.TruncatedTypes fe
+open import UF.Univalence
+
+module SizeTruncationandConnectedness
```

Comment:
These parameters should be part of the top-level module `UF.Size-TruncatedConnected`. There is no need to have yet another module (that you named `module SizeTruncationandConnectedness`) right after the top-level one.

Reply:
This is what I tried to say earlier, but I probably failed to express it clearly.

Reply:
I have to use funext first to import UF.TruncatedTypes so then I can use a paramter from general-truncations-exist. So I can't see a way around having two modules. Please describe how if possible

Reply:
Oh I got it! I just have to not give UF.TruncatedTypes the fe argument and provide it to general-truncations-exist instead.